views/site/login.php

```diff
@@ -24,10 +24,14 @@
         ],
     ]); ?>
 
+
```

Comment:
Why those spaces ?

Reply:
An error from myself, i had placed the password forgotten link there.